### PR TITLE
CDF attrs bug fix

### PR DESCRIPTION
**Pull request by tech3371**
# Change Summary
added `File_naming_convention` to global attribute and fixed data_type in all instruments.

### CODE REVIEW

**Review by maxinelasp — APPROVED** [review]
LGTM, comments are clear

File: imap_processing/mag/mag_cdf_attrs.py [hunk 1]
```diff
@@ -44,15 +44,15 @@
...
+    "L1B_SCI>Level-1B Science Data",
     # TODO: replace "sci" with descriptor "norm" / "burst"
     logical_source="imap_mag_l1b_sci",
     logical_source_desc="IMAP Mission MAG Instrument Level-1B Data.",
     instrument_base=mag_base,
 )
 
 mag_l1c_attrs = GlobalDataLevelAttrs(
-    "L1A>Level-1C",
+    "L1A_SCI>Level-1C Science Data",
```
Comment (vmartinez-cu) [inline imap_processing/mag/mag_cdf_attrs.py]: Should "L1A_SCI" be "L1C_SCI" for the l1c attrs? Just looking at l1b attrs for comparison, but I could be missing something important.

Comment (tech3371) [reply imap_processing/mag/mag_cdf_attrs.py]: good catch. Yes. it should be.

**Review by sdhoyt — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**